Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -8,13 +8,19 @@ public protocol ProjectPamphletViewModelInputs {
   /// Call with the project given to the view controller.
   func configureWith(projectOrParam: Either<Project, Param>, refTag: RefTag?)
 
+  /// Call after view loads and pass the parentViewController as parameter.
+  func parent(viewController: UIViewController?)
+
   /// Call when the view loads.
   func viewDidLoad()
 
   func viewDidAppear(animated: Bool)
 
   /// Call when the view will appear, and pass the animated parameter.
   func viewWillAppear(animated: Bool)
+
+  /// Call when the view will transition to a new trait collection.
+  func willTransitionToNewCollection(to newCollection: UITraitCollection)
```

Comment:
Think this should be `willTransition(toNewCollection collection: UITraitCollection)`

Reply:
Good catch!